### ISSUE TITLE
Separated tabs appear joined

### ISSUE THREAD

**Issue by pongloongyeat** [elementary/stylesheet#890]
## What Happened
When opening multiple tabs, tabs that are out of focus appear joined (no outline to discern between the tabs).

![Screenshot from 2020-11-23 06 41 50](https://user-images.githubusercontent.com/31680656/99919200-fffadc80-2d56-11eb-8c6d-d04420e6b1fa.png)

One can probably tell that the tabs are separated by the folder name label, but this might annoy others when trying to close tabs since the label is in the centre of the tab, which makes it hard for the user to tell how far the 'x' button is from the label. Can't get used to this either since this distance is not static (reduces with increasing number of open tabs) :/

## Platform Information
![Screenshot from 2020-11-23 08 51 14](https://user-images.githubusercontent.com/31680656/99922027-16119880-2d69-11eb-92e7-b17346d5d990.png)

**Comment by cassidyjames** [comment]
Yeah, I agree these could be better, especially since we're hiding the close button. I wonder if a small stroke separating tabs would be enough without making it that much more visually noisy:

![Screenshot from 2020-11-25 13-36-50](https://user-images.githubusercontent.com/611168/100279230-5d438900-2f23-11eb-92ec-6ae0449ca94f.png)

The difficult thing is dealing with not having the strokes on the start of the first one, then end of the last tab, _and_ not on the sides flanking and active tab.

**Comment by hanaral** [comment]
>Yeah, I agree these could be better, especially since we're hiding the close button. I wonder if a small stroke separating tabs would be enough without making it that much more visually noisy: ...

@cassidyjames what about making the tab’s width a little shorter - that way the highlight that appears on hover would no longer look conjoined with the active tab

**Comment by danirabbit** [comment]
These days we're using Adw.TabBar and there's a separator between unfocused tabs
